pipeline-scripts/UnlockTFstateIfLocked.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
can we not use capitals in the file name? locks weird,

`unlock-tf-state.sh` should be enough

Reply:
added changed to latest commit